index.html

```diff
@@ -580,7 +676,7 @@ <h4>Automation via Workflows</h4>
       "interact": {
         "service": [{
           "type": "UnmediatedPresentationService2021",
-          "serviceEndpoint": "https://example.edu/active-flows/123456"
+          "serviceEndpoint": "https://example.edu/workflows/123456/presentations"
```

Comment:
I don't hate the way this looks, but I worry about collisions on workflow `id` and its use in other systems.

Prefer for this endpoint and its parameters to be defined fully in this repo.

Reply:
It is fully defined here (in this PR): https://github.com/w3c-ccg/vc-api/pull/255/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R861